host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -2103,6 +2392,14 @@ int main(int argc, const char **argv)
       destroy_camera_component(&state);
       exit_code = EX_SOFTWARE;
    }
+   else if (state.raw_output && (status = create_splitter_component(&state)) != MMAL_SUCCESS)
```

Comment:
Build warning off this when cross-compiling:
userland/host_applications/linux/apps/raspicam/RaspiVid.c: In function ‘main’:
userland/host_applications/linux/apps/raspicam/RaspiVid.c:2412:30: warning: ‘status’ may be used uninitialized in this function [-Wmaybe-uninitialized]
    else if (state.raw_output && (status = create_splitter_component(&state)) != MMAL_SUCCESS)
The error isn't this line but the fact that create_splitter_component can return an uninitialised value - see above.